assets/doc/targets/neovim.md

```diff
@@ -15,30 +20,67 @@ To use the terminal's PID as input instead of Neovim's internal job-id of the te
 let g:slime_input_pid=1
 ```
 
-On Windows Neovim assigns a PID as well.
+PIDs of processes of potential target terminals are visible to Neovim on Windows as well as MacOS and Linux.
+
+
+
+
+##### Process Identification
 
 To manually check the right value of `job-id`  (but not `PID`) try:
 
 ```vim
-    echo &channel
+echo &channel
 ```
 
 from the buffer running your terminal.
 
 Another way to easily see the `PID` and job ID is to override the status bar of terminals to show the job id and PID.
 
 ```vim
- autocmd TermOpen * setlocal statusline=%{bufname()}%=id:\ %{b:terminal_job_id}\ pid:\ %{b:terminal_job_pid}
+autocmd TermOpen * setlocal statusline=%{bufname()}%=id:\ %{&channel}\ pid:\ %{jobpid(&channel)}
 ```
 
 See `h:statusline` in Vim's documentiation for more details.
 
-If you are using a plugin to manage your status line, see that plugin's documentation to see how to confiugre the status line to display `b:terminal_job_id` and `b:terminal_job_pid`.
+If you are using a plugin to manage your status line, see that plugin's documentation to see how to confiugre the status line to display `&channel` and `jobpid(&channel)`.
+
+Many status line plugins for Neovim are configured in lua.
+
+A useful Lua function to return the Job ID of a terminal is:
+
+```lua
+local function get_chan_jobid()
+	return vim.api.nvim_eval('&channel > 0 ? &channel : ""')
```

Comment:
that's a lot of margin space here (same at L62)

Reply:
fixed